### ISSUE TITLE
Synchronize options didn't work

### ISSUE THREAD

**Issue by iouzzrj** [gildas-lormeau/SingleFile#1011]
**Describe the bug**
Synchronize options didn't work as I expected.

**To Reproduce**
1. Install SingleFile extension from Microsofot Store.
2. Under Options, change File name template from the default settings to something like "{page-title}.html".
3. Check synchronize options.
4. Delete SingleFile extension.
5. Install SingleFile extension from Microsofot Store, again.
6. Check synchronize options.
7. Under Options, File name template is still the default value, not the last value I modified, like "{page-title}.html".

**Expected behavior**
If I have checked the synchronize options, then after a delete and reinstall cycle, the File name template should be the last value I modified, not the default one.

**Environment**
 - OS: Win10 home
 - Browser: MS Edge
 - Version: SingleFile 1.21.18

**Additional context**
I don't know if my understanding of synchronize options is correct...

**Comment by gildas-lormeau** [comment]
Actually, I don't know how the synchronization works in your example. The purpose of this feature is to share options between multiple browser instances. In step 4, you don't have a browser with SingleFile installed, so it's possible that Microsoft would delete the data at that point. For your information, SingleFile uses this API to implement synchronization: https://developer.chrome.com/docs/extensions/reference/storage/#property-sync. As you can see, the documentation is very light.

**Comment by iouzzrj** [comment]
Hi Gildas,
Thanks for clarifying. I don't know the API thing. So can synchronization works between two computer with MS Edge and SingleFile extension? I remember that I installed SingleFile on my laptop and changed the File name template. I also checked the synchronize options. However on my newly installed Win10 desktop, with synchronize options on, the File name template value is still the default one. Does these two computer have to be powered on at the same time for the synchronization to work?
